dev-libs/libixion/libixion-9999.ebuild

```diff
@@ -14,39 +14,45 @@ if [[ ${PV} == 9999 ]]; then
 	inherit git-r3 autotools
 else
 	SRC_URI="https://kohei.us/files/ixion/src/${P}.tar.xz"
-	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+	# depends on spdlog keywording: https://bugs.gentoo.org/708910
+	#KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+	KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~x86"
+
```

Comment:
no need for this, that's what keywording bugs are for. unnecessary empty line.

Reply:
I dropped the two comment lines locally.